.github/CONTRIBUTION.md

```diff
@@ -64,19 +66,20 @@ You can skip git commit hooks with:
 
 ## Releasing
 
-Currently only admins are able to create a release. A release consists of the following:
+Currently only admins are able to create a release, since you have to **commit and push straight to main branch** for the release script to run on Circle CI.
+A release consists of the following:
 
 - Bumping the package version based on your commits
 - Updating the API specification
 - Creating a new commit with the changed files
 - Creating a tag with the new version
-- Pushing the release commit and tag to master
+- Pushing the release commit and tag to main
 
 ### Step-By-Step
 
-1. Check out master and run `git pull`.
-1. Run `git clean -dfx && yarn` to make sure depenencies are up-to-date.
-1. Run `npm version [major | minor | patch] -m "chore(release): v%s"`. Use semver string based on conventional commits since last release. Ex: `npm version patch -m "chore(release): v%s"`.
+1. Check out main and run `git pull`.
+1. Run `git clean -dfx && yarn` to make sure dependencies are up-to-date.
+1. Run `npm version [major | minor | patch] -m "chore(release): v%s"`. Use a [semVer string based on conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) since last release. Ex: `npm version patch -m "chore(release): v%s"`.
```

Comment:
You might wanna add a hint about major/minor... if you think it helps

Reply:
that is what the link is for

Reply:
It is basically standard practice, so I don't think we have to explain it in our docs when it is already well documented